db/sqitch.conf

```diff
@@ -0,0 +1,8 @@
+[core]
```

Comment:
What's this change about?  I don't remember ``sqitch.conf`` getting changed the last few times we did a db migration.  Or maybe it did and I was super-lax about reading through the ``sqitch`` changes.